java/lib/src/main/java/com/svix/Endpoint.java

```diff
@@ -128,4 +129,20 @@ public EndpointStats getStats(final String appId, final String endpointId) throw
 			throw Utils.wrapInternalApiException(e);
 		}
 	}
+
+	public void replay(final String appId, final String endpointId, final ReplayIn replayIn) throws ApiException {
+		try {
```

Comment:
Just looking at how `recover` is implemented, shouldn't this be
```java
this.replay(appId, endpointId, replayIn, new PostOptions())
```

Reply:
Good catch, fix'd